content/zh/docs/concepts/architecture/garbage-collection.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Garbage Collection
+content_type: concept
+weight: 50
+---
+
+<!-- overview -->
+{{<glossary_definition term_id="garbage-collection" length="short">}} 这
+允许像下面这样清理资源:
+
+  * [Failed pods](/docs/concepts/workloads/pods/pod-lifecycle/#pod-garbage-collection)
+  * [Completed Jobs](/docs/concepts/workloads/controllers/ttlafterfinished/)
+  * [Objects without owner references](#拥有者)
+  * [Unused containers and container images](#容器镜像)
+  * [Dynamically provisioned PersistentVolumes with a StorageClass reclaim policy of Delete](/docs/concepts/storage/persistent-volumes/#delete)
```

Comment:
Where Chinese pages exist, eg https://kubernetes.io/zh/docs/concepts/workloads/pods/pod-lifecycle/, I would link to those.

Reply:
> 

I have modified it